qa/tox.ini

```diff
@@ -10,7 +10,15 @@ commands=flake8 --select=F,E9 --exclude=venv,.tox
 
 [testenv:mypy]
 basepython = python3
-deps = mypy==0.790
+deps =
+  mypy==0.901
+  types-boto
+  types-requests
+  types-jwt
+  types-paramiko
+  types-PyYAML
+  types-cryptography
+  types-python-dateutil
```

Comment:
I wonder if it'd make sense to have a separate `requirements-typing.txt` for each sub-project, or even a common one for all (in order to ensure as many type definitions as possible are always installed and used).

Reply:
#41859